models/staging/adwords/adwords_gclid_mapping.sql

```diff
@@ -0,0 +1,30 @@
+with clicks as (
+
```

Comment:
this model needs a test

Reply:
there are 4000+ null `gclids`

Reply:
this is fine - we did discover that the four columns (criteria_id, ad_group_id, adwords_campaign, gclid) can create a unique_id if we want this down the road